packages/tsconfig-reference/copy/ja/options/locale.md

```diff
@@ -0,0 +1,26 @@
+---
```

Comment:
@orta

I found this file is never listed but I did not investigate why . I'll send another PR like #335 .

Reply:
I think I hid this on purpose because either setting it in the tsconfig didn't work - or my the twoslash didn't work with these messages and so I couldn't prove them